### ISSUE TITLE
Consider suppressing gpg permission dialog or increasing timeout

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-workstation#154]
The workstation currently triggers a gpg key access permission dialog with a short timeout when decrypting data. Frequent dialogs like this are likely to cause alert fatigue -- users just click "Yes" without scrutinizing the context. In the event of malicious code execution, it's doubtful that the user would know _not_ to grant permission that one time.

![gpg dialog](https://raw.githubusercontent.com/freedomofpress/securedrop-workstation/master/docs/images/step3-decrypt-cropped.png)

We should consider increasing the timeout so that the dialog appears only once per boot, or suppressing it altogether, unless there are security benefits that are not clear. Note that this dialog provides no protection against workstation compromise, as it requires no further authorization.

**Comment by zenmonkeykstop** [comment]
This definitely needs doing - if the recommended workflow is to log out and/or shut down the workstation between sessions, I would agree with making it as long as possible. But it should definitely be as longer than 5 mins. An hour strikes me as a good minimum value.

**Comment by ninavizz** [comment]
@zenmonkeykstop Could we increase it from 1 hour to 8 or 12 hours? Only asking, because biggy files can take +1 hours to download, and updates may take a similar amount of time to process. Curious how this may impact the threat-model or if it would violate it. Thoughts/compromises @emkll or @redshiftzero?

**Comment by conorsch** [comment]
Big fan of removing the prompt entirely, but not sure we have the ability to do so easily, given the splitgpg tooling. The [splitgpg docs](https://www.qubes-os.org/doc/split-gpg/#setting-up-the-gpg-backend-domain) provide recommendations on lengthening the window:

```
[user@work-gpg ~]$ echo "export QUBES_GPG_AUTOACCEPT=86400" >> ~/.bash_profile
```

Let's add the equivalent file management targeting `sd-gpg` to the Salt config; I believe @zenmonkeykstop is already working on this.

**Comment by redshiftzero** [comment]
i definitely think we can make it 8 hours (same time as the API token timeout)

**Comment by conorsch** [comment]
Resolved via #207.
